e2e/pageobjects/ide/Ide.ts

```diff
@@ -40,6 +40,10 @@ export class Ide {
         await this.driverHelper.waitAndSwitchToFrame(By.css(Ide.IDE_IFRAME_CSS), timeout);
     }
 
+    async sleep(timeout: number) {
+        await this.driverHelper.wait(timeout);
```

Comment:
Please, remove this method and use "DriverHelper.wait()" 
example ```driverHelper.wait(1000)```

Reply:
has been removed from the test and PageObject